__tests__/endpoints.test.js

```diff
@@ -52,7 +52,37 @@ describe("/api/articles", () => {
   });
 });
 
-describe("api/nonsense", () => {
+describe.only("/api/articles/:article_id", () => {
+  test("GET:200, responds with an array of articles matching given article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.length).not.toBe(0);
+        body.article.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: 1,
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("GET: 404, sends an appropriate error message when given an invalid article_id", () => {
+    return request(app)
+      .get("/api/articles/99999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("article not found!");
+      });
+  });
```

Comment:
One test missing here. We need to test for an invalid id ('/api/articles/not-a-valid-id') return 400 'bad request'